Dockerfile

```diff
@@ -0,0 +1,40 @@
+# We need this to use GPUs inside the container
+FROM nvidia/cuda:11.2.2-base
+# Using a multi-stage build simplifies the s3prl installation
+# TODO: Find a slimmer base image that also "just works"
+FROM tiangolo/uvicorn-gunicorn:python3.8
+
+RUN apt-get update --fix-missing && apt-get install -y wget \
+    libsndfile1 \
+    sox \
+    git \
+    git-lfs
+
+RUN python -m pip install --upgrade pip
+RUN python -m pip --no-cache-dir install fairseq@git+https://github.com//pytorch/fairseq.git@f2146bdc7abf293186de9449bfa2272775e39e1d#egg=fairseq
+# TODO: Update URL if we merge back to s3prl repository
+RUN python -m pip --no-cache-dir install git+https://github.com/huggingface/s3prl.git@huggingface2#egg=s3prl
+
+COPY s3prl/ /app/s3prl
+COPY src/ /app/src
+
+# Setup filesystem
+RUN mkdir /app/data
+
+# Configure Git
+# TODO: Create a dedicated SUPERB account for the project?
+RUN git config --global user.email "lewis@huggingface.co"
+RUN git config --global user.name "SUPERB Admin"
+
+# Default args for fine-tuning
+ENV upstream_model osanseviero/hubert_base
+ENV downstream_task asr
+ENV hub huggingface
+ENV push_to_hf_hub True
+ENV override defaults
+
+WORKDIR /app/s3prl
+# Each task's config.yaml is used to set all the training parameters, but can be overridden with the `override` argument
+# The results of each training run are stored in /app/s3prl/result/downstream/{downstream_task}
+# and pushed to the Hugging Face Hub with name {hf_username}/superb-s3prl-{upstream_model}-{downstream_task}-uuid
+CMD python run_downstream.py -n ${downstream_task} -m train -u ${upstream_model} -d ${downstream_task} --hub ${hub} --push_to_hf_hub ${push_to_hf_hub} --override ${override}
```

Comment:
@leo19941227 I've now added the `override` argument to the `docker run` command and added a simple example in the README.

I had some trouble setting an empty string as the default value (for some reason this was triggering the `override` in the Docker container), so opted for a `defaults` variable that I check against in `run_downstream.py`. If this is too hacky for you, I'll try to find a better solution.

cc @osanseviero

Reply:
Hi @lewtun,
I am good with a default string. I am thinking that I would prefer we use the string "None" instead of "defaults" :)

Reply:
ok sounds good, i'll make the change later today!